Library/Homebrew/dependency_collector.rb

```diff
@@ -18,7 +18,7 @@
 class DependencyCollector
   # Define the languages that we can handle as external dependencies.
   LANGUAGE_MODULES = Set[
-    :chicken, :jruby, :lua, :node, :ocaml, :perl, :python, :python3, :rbx, :ruby
+    :lua, :perl, :python, :python3, :ruby
```

Comment:
Garg. Bad Dom.

Yeah, it's a relatively recent addition. There was a big Issue 6 or so months ago with "Homebrew supports Lua terribly" which was a fair comment, so I've been extremely slowly working on improving that (Though not a priority of mine, tbh).

The addition here was essentially a workaround for people who wanted it whilst investigated the possibility of whether we could do something similar to how we handle Python these days.